src/app/ui/overrides/adf-document-list.theme.scss

```diff
@@ -76,10 +50,12 @@
 
     .adf-datatable-cell-header,
     .adf-datatable-cell {
-      padding: 0 2px;
-      box-sizing: border-box;
       min-width: 120px;
 
+      &.adf-no-grow-cell {
+        min-width: 120px;
```

Comment:
not app-specific style, should be in ADF

Reply:
true, all these styles will be moved on ADF side with issue [ACA-2361](https://issues.alfresco.com/jira/browse/ACA-2361)